server/lib/src/server/mod.rs

```diff
@@ -889,6 +893,10 @@ impl<'a> QueryServerTransaction<'a> for QueryServerReadTransaction<'a> {
     fn get_domain_display_name(&self) -> &str {
         &self.d_info.d_display
     }
+
+    fn get_txn_cid(&self) -> &Cid {
```

Comment:
No todos in production - plus this isn't needed on the *read* transaction anyway.